ui/components/app/edit-gas-fee-popover/network-status/network-status.js

```diff
@@ -10,13 +9,6 @@ import StatusSlider from './status-slider';
 
 const NetworkStatus = () => {
   const { gasFeeEstimates } = useGasFeeContext();
-  let estBaseFee = null;
-  if (gasFeeEstimates?.estimatedBaseFee) {
-    // estimatedBaseFee is not likely to be below 1, value .01 is used as test networks sometimes
-    // show have small values for it and more decimal places may cause UI to look broken.
-    estBaseFee = toBigNumber.dec(gasFeeEstimates?.estimatedBaseFee);
-    estBaseFee = estBaseFee.lessThan(0.01) ? 0.01 : estBaseFee.toFixed(2);
-  }
```

Comment:
Was there a reason we removed this logic?

Reply:
This was added in previous PR after me and @mcmire had discussion about estimates being too long to show on page. But then we found that it was wrapping well on modal and also it is only test networks that value is too long.